frontend/src/lib/WorkflowParser.ts

```diff
@@ -361,6 +362,13 @@ export default class WorkflowParser {
         key: pathParts.slice(1).join('/'),
         source: StorageService.HTTPS,
       };
+    } else if (strPath.startsWith('volume://')) {
+      const pathParts = strPath.substr('volume://'.length).split('/');
```

Comment:
nit: report an error when pathParts.length === 0?

Reply:
`pathParts.length` never be zero I think.  if `strPath` is `volume://volume-mount-name`, it works fine.